interactions/client/bot.py

```diff
@@ -153,7 +154,10 @@ async def __compare_sync(self, data: dict, pool: List[dict]) -> Tuple[bool, dict
         attrs: List[str] = [
             name
             for name in ApplicationCommand.__slots__
-            if not name.startswith("_") and not name.endswith("id") and name != "version"
+            if not name.startswith("_")
+            and not name.endswith("id")
+            and name != "version"
+            and name != "default_permission"
```

Comment:
Would this be better?
```suggestion
            and name not in {"version", "default_permission"}
```